src/cursodejava/EjercicioDoce.java

```diff
@@ -37,7 +37,7 @@ public static void main(String[] args) {
             }
             
         }
-        
+       
         System.out.println("");
         System.out.println("Vector ordenado creciente:");
```

Comment:
```suggestion
        System.out.printf("Vector ordenado creciente:\n");
```
Se podria usar tambien de esta forma